__tests__/app.test.js

```diff
@@ -98,24 +99,27 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
-// describe("returns array of articles with correct properties", () => {
-//   it("returns correct keys in articles", () => {
-//     return request(app)
-//       .get("/api/articles")
-//       .then((response) => {
-//         const {
-//           body: { articles },
-//         } = response;
-//        articles.forEach((article) => {
-//           expect(article).toHaveProperty("author");
-//           expect(article).toHaveProperty("title");
-//           expect(article).toHaveProperty("article_id");
-//           expect(article).toHaveProperty("topic");
-//           expect(article).toHaveProperty("created_at");
-//           expect(article).toHaveProperty("votes");
-//           expect(article).toHaveProperty("article_img_url");
-//           expect(article).toHaveProperty("comment_count");
-//         });
-//       });
-//   });
-// });
+  describe("GET /api/articles", () => {
+    it("return array of articles with following properties", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+          const {
+            body: { articles },
+          } = response;
+          expect(Array.isArray(articles)).toBe(true);
+          expect(articles).toBeSorted({ descending: true });
+          articles.forEach((article) => {
+            expect(article).toHaveProperty("author");
+            expect(article).toHaveProperty("title");
+            expect(article).toHaveProperty("article_id");
+            expect(article).toHaveProperty("topic");
+            expect(article).toHaveProperty("created_at");
+            expect(article).toHaveProperty("votes");
+            expect(article).toHaveProperty("article_img_url");
+            expect(article).toHaveProperty("comment_count");
+          });
```

Comment:
We should also be asserting against the typeof the values as well, not just the keys here.